s30020-NikiforKoseda/zadanie 1.txt

```diff
@@ -0,0 +1,90 @@
+1.Uruchom terminal.
+Konfiguracja PUTTY, logowanie na szuflandię i wszystko elegancko działa
+
+2.Wyświetl zawartość aktualnego katalogu.
+ls wpisane
+
+3.Sprawdź gdzie aktualnie się znajdujesz.
+pwd wpisane
+
+4.Przejdź do katalogu Pulpit za pomocą ścieżki względnej
+cd // wpisane
+
+5.Sprawdź gdzie jesteś.
+pwd ponownie wpisane
+
+6.Wyświetl zawartość bieżącego katalogu.
+ls ponownie wpisane
+
+7.Wyświetl zawartość swojego katalogu domowego bez przechodzenia do niego (ls z
+ odpowiednim parametrem).
+ls $home wpisane
+
+8.Przejdź do katalogu Obrazy (lub innego katalogu, który jest dostępny - podaj 
+wprost, że zmieniasz katalog na inny bo np. Obrazy nie są u Ciebie dostępne) w 
+twoim katalogu domowym za pomocą ścieżki względnej.
+cd media wpisane
+
+9,Sprawdź gdzie jesteś.
+pwd ponownie wpisane
```

Comment:
Cała ta seria poleceń bez wyniku oraz bez wniosków nie podoba mi się.
Rozumiem, że w technikum takie rzeczy były wykonywane i jest to tak oczywiste jak to, że oddychasz teraz powietrzem i robisz to nieświadomie.
Natomiast jak widać samo podanie słowa "wpisane" wcale nie implikuje tu wykonania zadania lub wykonania go poprawnie tudzież rzetelnie.
Zadanie 7 to pięknie pokazuje.
`cd $home` nie zadziała bo bash nie definiuje zmiennej `$home` tylko `$HOME` a jednak rozróżnia wielkość liter.

Na podstawie powyższego wolałbym byś uzupełnił rozwiązania o faktycznie wyjście uzyskane w terminalu.